curriculum/challenges/english/10-coding-interview-prep/rosetta-code/sorting-algorithms-bead-sort.md

```diff
@@ -8,14 +8,26 @@ dashedName: sorting-algorithmsbead-sort
 
 # --description--
 
-Sort an array of positive integers using the [Bead Sort Algorithm](https://en.wikipedia.org/wiki/Bead_sort).
+Sort an array of positive integers using the Bead Sort Algorithm.
 
 A *bead sort* is also known as a *gravity sort*.
 
-The algorithm has O(S), where S is the sum of the integers in the input set: Each bead is moved individually.
+The algorithm has $O(S)$.
+
+- $O$ is Big $O$ notation
+- $S$ is the sum of the integers in the input set: Each bead is moved individually.
```

Comment:
I feel like we should have a description of how the bead sort works here instead of this brief bullet point list

Reply:
Why do we have this anyway? I do not feel like it is relevant information for the challenge.

Reply:
I don't know, to be honest😅. I left a message in chat, hoping to get to know better, to add better description